util/execdetails/execdetails.go

```diff
@@ -326,6 +326,8 @@ type RuntimeStats struct {
 	consume int64
 	// executor return row count.
 	rows int64
+	// executor concurrency information
+	concurrency string
```

Comment:
Maybe it's better to set its type as int instead of the string.

Reply:
Because the ```hashAgg``` operator has two variables and the information displayed is not the same like other variables such as rows. So I think it is better to use string than int, what's your opinion? @lzmhhh123

Reply:
Can we format like this?

```golang
concurrency []struct{name string, number int}
```

Reply:
Or
```golang
concurrency map[string]int
```